### PR TITLE
updated readme and contributing.md file

### PR DESCRIPTION
**Pull request by haripriya9647**
## Relevant Issue
Issue #276  the issue here

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,13 +9,25 @@ Here are some important resources to get ready for contributing:
   * [Deploying Federated Learning at scale](https://arxiv.org/abs/1902.01046) explains how a scalable FL system can be built 
   * [NimbleEdge Tutorials](./docs) is an overview of the library and our architecture
   
-## Testing 
+There are lot of ways in which you can help us!!
```
Comment (divyanshugit) [inline CONTRIBUTING.md:12]: You can make this as main heading using #

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,13 +9,25 @@ Here are some important resources to get ready for contributing:
...
 
-The library is in the initial stages and we would like to build a complete suite of tests. Till then we can only test functionality 
+## Do you want to contribute to code ??
```
Comment (divyanshugit) [inline CONTRIBUTING.md:14]: Please use only single `question mark (?)`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,7 +40,19 @@ HTML files are generated under `build/html` directory. Open the `index.html` fil
...
+
+## Do you love graphic or UI/UX designing ??
+We are open to these contributions as well.We will be glad if someone comes up with a new UI/UX design for our NimbleEdge website so that we can work accordingly and make it look more attractive and user friendly.
 
+## Do you really love marketing ??
+We are social, in every sense of the word. And we love nothing more than building new relationships.To build relationships,we need people who
+can help us in reaching out to new contributors and spread a word about our EnvisEdge.
+
+## Is there any other way to contribute??
+Yayy!! contributions types are not only restricted to above listed .You can do it lot more ways.You can train new contributors in the community,open good first issues, speak about nimbleedge in meetups,conferences and standups,you can spread a word about community and lot more!!
```
Comment (divyanshugit) [inline CONTRIBUTING.md:54]: Please use proper the spaces and commas

File: README.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ NimbleEdge/EnvisEdge
 └── tests                                   <-- tests
 ```
 
-# QuickStart
+# Installation
```
Comment (divyanshugit) [inline README.md:66]: Quickstart is more apt over here. You can add installation section as subheadings before the installing code block

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -9,13 +9,25 @@ Here are some important resources to get ready for contributing:
...
+* We request you to use the autopep-8 reformat **on the VCS changes** before the final commit.
+* We expect you to ensure indentation consistency.
+* You must follow Google code style.
+* You must follow numpy style guides [here](https://numpydoc.readthedocs.io/en/latest/format.html).
+* You must not forget to include documentation of the code you are contributing to when adding a new module/feature to EnvisEdge.
+* Your pull request must pass all the checks and has no conflicts.
+* Your pull request must include detailed commit messages along with title.
 
-## Documentation
+## Do you want to test our code ?
```
Comment (divyanshugit) [inline CONTRIBUTING.md:25]:
```suggestion
## Do you want to test our code?
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -9,13 +9,25 @@ Here are some important resources to get ready for contributing:
...
 
-When adding a new module/feature to EnvisEdge, please make sure to add the documentation for it. The documentation source is stored under the [docs](./docs) directory and written in [reStructuredText format](http://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html).
+The library is in the initial stages and we would like to build a complete suite of tests. Till then we can only test functionality.If you are someone interested in testing our code please head towards our [discord](https://nimbleedge.ai/discord) channel to have a inital conversation with project maintainers.
+
+## Are you someone interested in writing documentation ?
```
Comment (divyanshugit) [inline CONTRIBUTING.md:29]: Please use proper identation throughout the file. You can use grammarly for resolving it.
```suggestion
## Are you someone interested in writing documentation?
```

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Pull request merged**